test_regress/t/t_concatenate_casts.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
Please rename t_concat_casts to match other test prefixes, then I think good to go.